datasets/us/ofac/enforcement_actions/crawler.py

```diff
@@ -0,0 +1,291 @@
+    if review is None:
+        prompt_result = run_typed_text_prompt(context, PROMPT, article_html, Defendants)
+        review = request_review(
+            context,
+            url,
+            article_html,
+            HTML,
+            "Enforcement Action Notice",
+            url,
+            prompt_result,
+            MODEL_VERSION,
+        )
+
+    if check_something_changed(context, review, article_html, article_element):
+        # In the first iteration, we're being super conservative and rejecting
+        # export if the source content has changed regardless of whether the
+        # extraction result has changed. If we see this happening and we see that
+        # the extraction result reliably identifies real data changes, we can
+        # relax this to only reject if the extraction result has changed.
+
+        # Similarly if we see that broad markup changes don't trigger massive
+        # re-reviews but legitimate changes are reliably detected, we can allow
+        # it to automatically request re-reviews upon extraction changes.
+        global something_changed
+        something_changed = True
+        return
+
+    if not review.accepted:
+        return
+
+    for item in review.extracted_data.defendants:
+        entity = context.make(item.entity_schema)
+        entity.id = context.make_id(item.name, item.address, item.country)
+        entity.add("name", item.name, origin=DEFAULT_MODEL)
+        if item.address != item.country:
+            entity.add("address", item.address, origin=DEFAULT_MODEL)
+        entity.add("country", item.country, origin=DEFAULT_MODEL)
+        entity.add("alias", item.aliases, origin=DEFAULT_MODEL)
+        entity.add("topics", "reg.action")
+
+        # We try to link press releases that refer to an original press release number
+        # back to the original press release by using that in the sanction key.
+
+        # In practice often the entity ID and thus sanction ID differs because of
+        # different levels of address details in the press releases.
+        sanction = h.make_sanction(context, entity)
+        h.apply_date(sanction, "date", date)
+        sanction.set("sourceUrl", url)
+        sanction.add("sourceUrl", item.pdf_url, origin=DEFAULT_MODEL)
+        sanction.add("status", item.status, origin=DEFAULT_MODEL)
+        sanction.add("summary", item.notes, origin=DEFAULT_MODEL)
+
+        for related_company in item.related_companies:
+            related_company_entity = make_related_company(context, related_company.name)
+            link = make_company_link(
+                context, entity, related_company_entity, related_company.relationship
+            )
+            context.emit(related_company_entity)
+            context.emit(link)
+
+        article = h.make_article(context, url, title=article_name, published_at=date)
+        documentation = h.make_documentation(context, entity, article, date=date)
```

Comment:
It can now take the date from the article

```suggestion
        documentation = h.make_documentation(context, entity, article)
```